tests/test_security_middleware.py

```
@@ -22,6 +22,17 @@
 import pytest
 from typing import Optional, Callable, Any
 
+from datetime import datetime
+import uuid
+
+import pytest
```

Ruff (0.12.2)

Redefinition of unused `pytest` from line 22

Remove definition: `pytest`

(F811)